lib/connection-pool.js

```diff
@@ -19,7 +19,7 @@ var httpOptionsBlob = []
 var lastUsed = 0
 
 // limit number of sockets
-http.globalAgent.maxSockets = 10
+http.globalAgent.maxSockets = 3
```

Comment:
Oh gosh, sorry all, I didn’t mean to update this commit.


Bob Ullery
CTO
bob@waysay.com <mailto:bob@waysay.com>
574-532-1412



> On Sep 8, 2016, at 7:01 PM, Martin Heidegger <notifications@github.com> wrote:
> 
> In lib/connection-pool.js <https://github.com/megastef/node-crate/pull/12#discussion_r78104746>:
> 
> > @@ -19,7 +19,7 @@ var httpOptionsBlob = []
> >  var lastUsed = 0
> >  
> >  // limit number of sockets
> > -http.globalAgent.maxSockets = 10
> > +http.globalAgent.maxSockets = 3
> I think this shouldn't be changed in THIS commit.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/megastef/node-crate/pull/12/files/1977c73e4a30fce16cdd14f257e66313f0a4028c#r78104746>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ANKbmDVRwB6c93Im2pUFTs-qCeQKN5kSks5qoJPkgaJpZM4JorPn>.
>